sdcm/cluster.py

```diff
@@ -1511,13 +1511,13 @@ def wait_for_nodes_up_and_normal(self, nodes):
     def get_scylla_version(self):
         if not self.nodes[0].scylla_version:
             try:
-                result = self.nodes[0].remoter.run("rpm -q {}".format(self.nodes[0].scylla_pkg()))
+                result = self.nodes[0].remoter.run("scylla --version")
```

Comment:
Is "scylla --version" always working at this stage?
We can try it and if not, try the "rpm -q"